src/Elasticsearch.Net/Configuration/RequestConfiguration.cs

```diff
@@ -170,6 +179,16 @@ public RequestConfigurationDescriptor RequestTimeout(TimeSpan requestTimeout)
 			return this;
 		}
 
+		/// <summary>
+		/// Associate an Id with this user-initiated task, such that it can be located in the cluster task list.
+		/// <pre>https://www.elastic.co/guide/en/elasticsearch/reference/6.2/tasks.html#_identifying_running_tasks</pre>
+		/// </summary>
+		public RequestConfigurationDescriptor OpaqueId(string opaqueId)

```

Comment:
Add something to indicate that this is valid only for Elasticsearch 6.2.0+

Reply:
++

Reply:
I think rather than having a link to docs, something like "Valid for Elasticsearch 6.2.0+" would be sufficient.